### DISCUSSION TITLE
章节名和正文距离相距太远，可以改吗？

### DISCUSSION THREAD

**Discussion by sastars9** [nju-lug/NJUThesis#183] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/128053942/225627990-b4054904-da35-46b3-92ce-0c3ee04665bd.png)
这里有很大空格，谢谢

**Comment by atxy-blip** [comment, marked as answer]
导言区加如下设置，数值可以自行调整。

 ```tex
\ctexset{
   chapter/beforeskip = 10pt,
   chapter/afterskip = 10pt
}
```

**Reply by DeclK** [reply]
afterskip 能够调节章节后面的空间，但是 beforeskip 似乎没有作用，没办法调节章节之前的空间

**Reply by atxy-blip** [reply]
显然 skip 数值可以是负的。
![image](https://github.com/nju-lug/NJUThesis/assets/63113546/f8d80fdc-8fd8-46cf-9d1c-990e6c3a5884)

**Comment by sastars9** [comment]
谢啦，已解决
